.changeset/clean-berries-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+'@guardian/source-foundations': major
+'@guardian/source-react-components': major
```

Comment:
Given there's no perceptible changes and this constitutes a refactor, should we consider these `patch` changes?

```suggestion
'@guardian/source-foundations': patch
'@guardian/source-react-components': patch
```